index.html

```diff
@@ -39,6 +39,80 @@ <h1>I'm José</br>
             </li>
         </ul>
     </section>
+    <section class="works">
```

Comment:
Great job team 🤝

- [ ] Kindly make sure the main section is aligned with `display: grid` and the cards inside are aligned with `display: flex`.